ui/src/pages/Logs/Logs.tsx

```diff
@@ -175,9 +198,40 @@ const logsDisplayStyle = {
   overflow: 'hidden',
   fontFamily: 'monospace',
 };
+
+const colorArray: string[] = [
+  purple[100],
+  red[100],
+  blue[100],
+  teal[100],
+  green[100],
+  cyan[100],
+  orange[100],
+  yellow[100],
+  brown[100],
+  blueGrey[100],
+];
+// Keeps track of what color to assign to each unique container
+let colorCounter = 0;
+// Containers object holds key value pairs of {containerName: containerColor}
+const containerTracker: Record<string, string> = {};
+let containerColor: string;
+let containerIconColor: string;
+
 function Row({ containerName, containerId, time, stream, log }: DockerLog) {
   const [open, setOpen] = useState<boolean>(false);
 
+  // Assigning color to container only if its a new container
+  if (!containerTracker.hasOwnProperty(containerName)) {
+    containerColor = colorArray[colorCounter];
```

Comment:
To avoid indexing out of the bounds of the array, you can do:
`array[counter % array.length]`

If the array length is 5 but the counter is 7, you will be indexing at 2.

The problem we have here with limited colors is a good use-case for this technique